src/sentry/conf/server.py

```diff
@@ -1513,6 +1513,7 @@ def create_partitioned_queues(name):
         # started up, only pulled and made available for `devserver` which will start
         # it with `devservices attach --is-devserver reverse_proxy`.
         "with_devserver": True,
+        "devserver_name": "proxy",
```

Comment:
I'd just rename the service so that `devservices rm proxy` works.

Reply:
The reason I gave the service a different name is so that it's clear in the devservices what the service is, To me the label in the devserver doesn't need to be as verbose.

I want to know this is a nginx proxy, I don't relaly want to drop `nginx` from the devservice name.

Reply:
Hmm, I think that's fair, this is for cosmetics only. Wdyt https://github.com/getsentry/sentry/pull/18948#discussion_r429456041 then?

Reply:
yeah I'm fine w/ that